File: TimeFlies/vc/todo/TodoCollectionViewCell.m
//
//  TodoCollectionViewCell.m
//  testTabbarController
//
//  Created by 林伟池 on 15/10/26.
//  Copyright © 2015年 林伟池. All rights reserved.
//

#import "TodoCollectionViewCell.h"

@implementation TodoCollectionViewCell

#pragma mark - view init
- (void)viewInitWith:(NSDate *)startTime Text:(NSString *)text
{
    self.myDetail.text = text;
    
    NSDateFormatter* dateFormatter = [[NSDateFormatter alloc] init];
    [dateFormatter setDateFormat:@"yyyy/MM/dd"];
    
    
    self.myStartTime.text = [dateFormatter stringFromDate:startTime];
    
//    NSLog(@"%@init", self);
//    [self layoutIfNeeded];
}
#pragma mark - ui

#pragma mark - delegate

#pragma mark - notify

@end

File: TimeFlies/vc/todo/TodoCollectionViewCell.swift
//
//  TodoCollectionViewCell.swift
//  TimeFlies
//
//  Created by 林伟池 on 16/4/8.
//  Copyright © 2016年 林伟池. All rights reserved.
//

import Foundation

class SwiftTodoCollectionViewCell: UICollectionViewCell {
    
    
    //MARK: property
    @IBOutlet var myStartTime:UILabel?
    @IBOutlet var myDetail:UITextView?
    
    //MARK: init
    
    func viewInitWith(_ startTime: Date, text: String) {
        myDetail?.text = text
        let dateFormatter = DateFormatter()
        dateFormatter.dateFormat = "yyyy/MM/dd"
        
        myStartTime?.text = dateFormatter.string(from: startTime)
    }
    
    
    //MARK: ui
    
    
    //MARK: delegate
    
    
    //MARK: other
    
}
